easybuild/easyconfigs/n/NAMD/NAMD-2.12-foss-2017a-mpi.eb

```diff
@@ -7,20 +7,22 @@ description = """NAMD is a parallel molecular dynamics code designed for high-pe
  large biomolecular systems."""
 
 toolchain = {'name': 'foss', 'version': '2017a'}
-toolchainopts = {'usempi': True, 'pic': True}
+toolchainopts = {'pic': True, 'usempi': True, 'openmp': True}
 
 sources = ['NAMD_%(version)s_Source.tar.gz']
 
+patches = [
+    'charm-remove-auto-from-fortran-args.patch',
```

Comment:
@akesandgren Since this is a patch to the CHARM included with NAMD 2.12, please rename the patch file to `NAMD-2.12_charm...`